daffodil-cli/src/it/resources/org/apache/daffodil/CLI/single.xsd

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
By convention the extension for DFDL schemas is ".dfdl.xsd"

Reply:
Updated